include/tsl/sparse_hash.h

```diff
@@ -1231,7 +1240,7 @@ class sparse_hash : private Allocator,
        * We can't use `vector(size_type count, const T& value, const Allocator&
        * alloc)` as it requires the value T to be copyable.
        */
-      m_sparse_buckets_data.resize(
+      m_sparse_buckets_data.resize( //TODO: this is the current critical point
```

Comment:
left over comment?

Reply:
You are completely correct. I removed it.